From 71795296fb37c448ee05c637e228c2af5468ad4f Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Thu, 9 Nov 2017 14:07:00 +0200
Subject: [PATCH 2867/5242] MLK-16918-12: arm64:dts: fsl-imx8qm-lpddr4-arm2:
 Update dts files for nwl-mipi-dsi bridge

commit  7b087320b38e5ee9e4dd17cd8127a96b24d34378 from
https://source.codeaurora.org/external/imx/linux-imx.git

Now, the NWL MIPI-DSI driver is implemented as a real bridge and uses
the "nwl,mipi-dsi" compatible.
This patch updates the ADV7535 and RM67191 specific DTS files to
8QXP-MEK.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts  |   16 ++++++++++++----
 .../dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts  |   16 ++++++++++++----
 2 files changed, 24 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts
index 2a03be3..06fe3a2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts
@@ -30,7 +30,7 @@
 
 		port {
 			adv7535_1_in: endpoint {
-				remote-endpoint = <&mipi_dsi1_out>;
+				remote-endpoint = <&mipi_dsi_bridge1_out>;
 			};
 		};
 	};
@@ -42,9 +42,13 @@
 
 &mipi_dsi1 {
 	status = "okay";
+};
+
+&mipi_dsi_bridge1 {
+	status = "okay";
 
 	port@1 {
-		mipi_dsi1_out: endpoint {
+		mipi_dsi_bridge1_out: endpoint {
 			remote-endpoint = <&adv7535_1_in>;
 		};
 	};
@@ -66,7 +70,7 @@
 
 		port {
 			adv7535_2_in: endpoint {
-				remote-endpoint = <&mipi_dsi2_out>;
+				remote-endpoint = <&mipi_dsi_bridge2_out>;
 			};
 		};
 	};
@@ -78,9 +82,13 @@
 
 &mipi_dsi2 {
 	status = "okay";
+};
+
+&mipi_dsi_bridge2 {
+	status = "okay";
 
 	port@1 {
-		mipi_dsi2_out: endpoint {
+		mipi_dsi_bridge2_out: endpoint {
 			remote-endpoint = <&adv7535_2_in>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts
index 052a420..4e0c3c0 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-rm67191.dts
@@ -20,6 +20,10 @@
 
 &mipi_dsi1 {
 	status = "okay";
+};
+
+&mipi_dsi_bridge1 {
+	status = "okay";
 
 	panel@0 {
 		compatible = "raydium,rm67191";
@@ -29,13 +33,13 @@
 		panel-height-mm = <121>;
 		port {
 			panel1_in: endpoint {
-				remote-endpoint = <&mipi1_out>;
+				remote-endpoint = <&mipi_bridge1_out>;
 			};
 		};
 	};
 
 	port@1 {
-		mipi1_out: endpoint {
+		mipi_bridge1_out: endpoint {
 			remote-endpoint = <&panel1_in>;
 		};
 	};
@@ -47,6 +51,10 @@
 
 &mipi_dsi2 {
 	status = "okay";
+};
+
+&mipi_dsi_bridge2 {
+	status = "okay";
 
 	panel@0 {
 		compatible = "raydium,rm67191";
@@ -56,13 +64,13 @@
 		panel-height-mm = <121>;
 		port {
 			panel2_in: endpoint {
-				remote-endpoint = <&mipi2_out>;
+				remote-endpoint = <&mipi_bridge2_out>;
 			};
 		};
 	};
 
 	port@1 {
-		mipi2_out: endpoint {
+		mipi_bridge2_out: endpoint {
 			remote-endpoint = <&panel2_in>;
 		};
 	};
-- 
1.7.9.5

